src/api/xxAMIDOxx.xxSTACKSxx.Domain/MenuAggregateRoot/Entities/Converters/DynamoDbCategoryConverter.cs

```diff
@@ -0,0 +1,49 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Amazon.DynamoDBv2.DataModel;
+using Amazon.DynamoDBv2.DocumentModel;
+using Newtonsoft.Json;
+using xxAMIDOxx.xxSTACKSxx.Domain.Entities;
+
+namespace xxAMIDOxx.xxSTACKSxx.Domain.MenuAggregateRoot.Entities.Converters
+{
+    // IMPORTANT!!!
+    // Since Categories is IReadOnlyCollection<T> we need the converter to work with that interface both during serialization
+    // and deserialization. If Categories was a property using a CONCRETE type like List<T>, then the DynamoSDK would automatically handle
+    // the collection mapping and we could've made a converter for only a single Category
+    // DynamoDB .NET Object Persistence Model Types - https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/DotNetSDKHighLevel.html#DotNetDynamoDBContext.SupportedTypes
+    public class DynamoDbCategoryConverter : IPropertyConverter
```

Comment:
Is this class something directly from AWS or would it be worth wrapping with some unit tests?